build.sbt

```diff
@@ -42,7 +42,7 @@ libraryDependencies ++= Seq(
   "io.circe" %% "circe-optics" % circeVersion,
   "io.circe" %% "circe-generic" % circeVersion,
   "io.circe" %% "circe-generic-extras" % circeVersion,
-  "uk.gov.nationalarchives" %% "consignment-api-db" % "0.0.25",
+  "uk.gov.nationalarchives" %% "consignment-api-db" % "0.0.26",
```

Comment:
There is a version 0.0.27 available for the `consignment-api-db`. Not sure you want to use this version instead?